src/modules/sensors/vehicle_air_data/VehicleAirData.cpp

```diff
@@ -319,6 +323,31 @@ void VehicleAirData::Run()
 	perf_end(_cycle_perf);
 }
 
+void VehicleAirData::UpdateRelativeCalibrations(const hrt_abstime time_now_us)
+{
+	// delay calibration to allow all drivers to start up
+	_calibration_t_first = _calibration_t_first == 0 ? time_now_us : _calibration_t_first;
```

Comment:
The ternary operator forces you to have an `else` part, which is unwanted here, what about this?
```
if (_calibration_t_first == 0) {
    _calibration_t_first = time_now_us;
}
```